### ISSUE TITLE
Can't reproduce synthetic experiments!

### ISSUE THREAD

**Issue by antoniocollesei** [MehrdadMansouri/Aristotle#1]
Hello,

I am trying to reproduce your Aristotle code.
I understand that real data are not available publicly, so I am trying to run the Synthetic_Simulation.m code, but I am actually having a few issues:
- **Synthetic_Simulation.m** generates Outcome9, Features9, Pathway9 text files; they look like the outcome vector R, the matrix of omics profiles G and the prior knowledge about pathways: what about confounders?;
- the next step would be **Synthetic_Filtering.m** but it requires files .mat that are not really available;
- I don't see column names in the synthetic data files I generated, are they supposed to be there?

In conclusion, am I missing something? The code, as it is, does not look like it can be reproduced without interventions!
Looking forward to your kind answer...
Thank you,
Antonio
